tutorials/cpp/sensor-modeling/Dockerfile

```diff
@@ -1,11 +1,4 @@
-################################################################################################################################################################
-
-# @project        Open Space Toolkit ▸ Astrodynamics
-# @file           tutorials/cpp/sensor-modeling/Dockerfile
-# @author         Lucas Brémond <lucas@loftorbital.com>
-# @license        Apache License 2.0
-
-################################################################################################################################################################
+# Apache License 2.0 
 
 FROM openspacecollective/open-space-toolkit-base:0.1.4
```

Comment:
Update this part? Do we even want to keep these in tutorials/cpp?

Reply:
Eh might as well. No harm.